relayer/spy_relayer/src/relayer/solana.ts

```diff
@@ -44,6 +58,28 @@ export async function relaySolana(
 
   const keypair = Keypair.fromSecretKey(walletPrivateKey);
   const payerAddress = keypair.publicKey.toString();
+  const anchorWallet = new AnchorWallet(keypair);
+  const solanaProvider = new AnchorProvider(connection, anchorWallet, { commitment: "confirmed" });
+  const env = getRelayerEnvironment();
+
+  const swimUsdMint = env.swimUsdMint;
+  const solanaRoutingContract = new Program(
+    idl.propeller,
+    env.swimSolanaContractAddress,
+    solanaProvider,
+  );
+
+  const twoPoolProgram = new Program(
+    idl.twoPool,
+    env.swimTwoPoolAddress,
+    solanaProvider
+  );
+
+  const propellerEngineAnchorProvider = new AnchorProvider(
```

Comment:
Why do you need a second provider?

Reply:
this is unnnecessary. he was probably just copying from my engine.test.ts setup

Reply:
good catch, removed